src/main/kotlin/gg/essential/essential-loader.gradle.kts

```diff
@@ -0,0 +1,64 @@
+package gg.essential
+
+import gg.essential.gradle.multiversion.Platform
+import gg.essential.gradle.util.RelocationTransform.Companion.registerRelocationAttribute
+
+
+plugins {
+    id("gg.essential.loom")
+}
+
+val platform = Platform.of(project)
+
+val essentialLoader: Configuration by configurations.creating
+
+when {
+    platform.isLegacyForge -> {
+        dependencies {
+            "runtimeOnly"(essentialLoader("gg.essential:loader-launchwrapper:1.1.3")!!)
```

Comment:
Upon further consideration, I believe we do want to use `implementation` here instead of `runtimeOnly`. Only legacy forge you can only declare a single tweaker class, so if the mod has a tweaker of its own, then that tweaker needs to extend the essential loader tweaker (which is something loader does explicitly support) for that to get a chance to run.